pages/volunteer/Volunteer.js

```diff
@@ -0,0 +1,122 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppiaFoundationWebsite.controller('volunteerPage', [
+  '$scope', 'VOLUNTEER_INFO', function($scope, VOLUNTEER_INFO) {
+    var designSlideImages = [];
+    var designSlideQuotes = [];
+    var designSlideNames = [];
+    var designSlideJobTitles = [];
+    VOLUNTEER_INFO.design.forEach(function(designerProfile) {
+      designSlideImages.push(designerProfile.profilePictureImageFilename);
+      designSlideQuotes.push(designerProfile.professionalQuote);
+      designSlideNames.push(designerProfile.fullName);
+      designSlideJobTitles.push(designerProfile.jobTitle);
+    });
+    $scope.designSlideQuotes = designSlideQuotes;
```

Comment:
No need for this. Just declare as $scope.designSlideQuotes to start with, etc. Ditto below.

In fact I would go further and suggest:

    $scope.designSlides = [];
    VOLUNTEER_INFO.design.forEach(function(designerProfile) {
      $scope.designSlides.push({
        imageUrl: ...,
        quote: ...,
      });
    });

and similar elsewhere. That should make your code a lot shorter.

Reply:
I was trying to avoid creating 4 big array of dicts and roll with 16 arrays. I will refactor accordingly.